apps/src/StudioApp.js

```diff
@@ -284,8 +294,8 @@ StudioApp.prototype.init = function(config) {
       }
     };
     // Depends on ResizeSensor.js
-    var ResizeSensor = require('./ResizeSensor');
-    new ResizeSensor(document.getElementById('visualizationColumn'), resize);
+    var attachResizeSensor = require('./ResizeSensor');
+    attachResizeSensor(document.getElementById('visualizationColumn'), resize);
```

Comment:
This was caught by the "use constructor for side effects" check, note how we called `new ResizeSensor(...)` but never assigned the result to anything.  I dug into ResizeSensor and it's not really a class at all; just a function that has a bunch of internal functions and hooks up some handlers/callbacks.  So my solution here is to treat ResizeSensor.js' export as a function, not as a class.